assignments/objects.js

```diff
@@ -11,54 +11,176 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+	id: 0,
+	name: 'Example',
+	email: 'examples@you.edu',
+	gender: 'F',
+};
 
 // Write your intern objects here:
 
+/* function Intern(id, name, email, gender) {
+	this.id = id,
+	this.name = name,
+	this.email = email,
+	this.gender = gender
+}
 
-// ==== Challenge 2: Reading Object Data ==== 
+const lavon = new Intern(6,'Lavon','tenees.lavon@gmail.com','F'); */
+
+const mitzi = {
+	id: 1,
+	name: 'Mitzi',
+	email: 'mmelloy0@psu.edu',
+	gender: 'F',
+};
+
+const kennan = {
+	id: 2,
+	name: 'Kennan',
+	email: 'kdiben1@tinypic.com',
+	gender: 'M',
+};
+
+const keven = {
+	id: 3,
+	name: 'Keven',
+	email: 'kmummery2@wikimedia.org',
+	gender: 'M',
+};
+
+const gannie = {
+	id: 4,
+	name: 'Gannie',
+	email: 'gmartinson3@illinois.edu',
+	gender: 'M',
+};
+
+const antonietta = {
+	id: 5,
+	name: 'Antonietta',
+	email: 'adaine5@samsung.com',
+	gender: 'F',
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(mitzi.name);
+
 // Kennan's ID
 
+console.log(kennan.id);
+
 // Keven's email
 
+console.log(keven.email);
+
 // Gannie's name
 
+console.log(gannie.name);
+
 // Antonietta's Gender
+console.log(antonietta.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
 
-// Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+const kennan = {
+	id: 2,
+	name: 'Kennan',
+	email: 'kdiben1@tinypic.com',
+	gender: 'M',
+	speak: function() {
```

Comment:
This is correct for declaring a method on an object at the same time as the object is being created. How would you add a method to an object that has already been declared? DM if you want some help reviewing this.